src/tests/ial_verification.py

```diff
@@ -538,19 +539,42 @@ def peripheral_sending_and_receiving_unframed_empty_pdu_llid_0b01_cis(transport,
     if not initiator:
         return success
 
+    start_pkt_seq_num_P_To_C = 4
+    start_pkt_seq_num_C_To_P = 4
+    pkt_seq_num_P_To_C = 0
+    pkt_seq_num_C_To_P = 0
+
+    SDU_Interval_C_To_P_ms = params.SDU_Interval_C_To_P // 1000
```

Comment:
I think that converting the SDU interval to integral milliseconds (`//`) would give trouble with intervals like 7.5ms. `SDU_Interval_C_To_P_ms` should probably be a floating-point number, or the conversion should be rolled into the sequence number calculation such that truncation is avoided.

Reply:
Done now.